jekyll/_cci2/custom-images.md

```diff
@@ -16,10 +16,28 @@ This document describes how to create and use custom Docker images with CircleCI
 
 ## Overview
 
-CircleCI 2.0 gives you access to the power and flexibility of Docker. One of the ways you can take advantage of this is to create custom Docker images for your jobs. Following are the benefits of creating a custom image:
+Docker is a first-class citizen in CircleCI
+and provides you with a powerful way
+to specify dependencies for your projects.
+If CircleCI's [convenience images]({{ site.baseurl }}/2.0/circleci-images/) do not suit your needs,
```

Comment:
No possessive on CircleCI (it is a product name and company name)-> the CircleCI convenience images

Reply:
Yep, great catch!